limbo/limbo.py

```diff
@@ -162,6 +173,7 @@ def init_config():
     getif(config, "logfile", "LIMBO_LOGFILE")
     getif(config, "logformat", "LIMBO_LOGFORMAT")
     getif(config, "plugins", "LIMBO_PLUGINS")
+    getif(config, "needmention", "LIMBO_NEEDMENTION")
```

Comment:
Okay for now, but eventually we should propose to support a configuration file in addition to (or instead of) environment variables.